stanford/roles/cluster/defaults/main.yml

```diff
@@ -125,6 +130,11 @@ CLUSTER_SECURITY_GROUP_RULES:
       from_port: 22
       to_port: 22
       cidr_ip: 0.0.0.0/0
+  coursegraph:
+    - proto: tcp
+      from_port: 22
+      to_port: 22
+      cidr_ip: "{{ DEPLOYMENT_CIDR }}"
```

Comment:
We had originally drafted this with port 7474 open too, but since we're just going in via SSH, I think this should be enough.